### ISSUE TITLE
react-native init throws an error:  Cannot find module './clone.js'

### ISSUE THREAD

**Issue by gran33** [facebook/react-native#22088]
## Environment
No output.
But I got this in old RN project:
```
Environment:
  OS: macOS High Sierra 10.13.3
  Node: 8.11.1
  Yarn: 0.24.5
  npm: 6.4.1
  Watchman: 4.9.0
  Xcode: Xcode 9.3.1 Build version 9E501
  Android Studio: 3.1 AI-173.4819257

Packages: (wanted => installed)
  react: NO RELEVANT
  react-native: NO RELEVANT
```
## Description
`react-native init <PROJECT_NAME>` failed with this error:
```
Error: Cannot find module './clone.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/<PATH_TO_PROJECT>/node_modules/graceful-fs/graceful-fs.js:4:13)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```
## Reproducible Demo
Simply try to create new project:
`react-native init <PROJECT_NAME>`

**Comment by alexsmartens** [comment]
I am having the same problem!

  OS: Win 10
  Node: 10.13.0 
  npm: 6.4.1

**Comment by gran33** [comment]
Smell to me like the problem is in `graceful-fs` and the fix is here:
https://github.com/isaacs/node-graceful-fs/commit/a1587eae7c5dcf74ccf6556ac7d6f704ea794804

Same old issues, our environment is soo flaky and can be broken so easily :(

**Comment by gran33** [comment]
It's working now.

**Comment by alexsmartens** [comment]
This is what helped me:
```
npm cache clean -f
npm -g install npm
```

**Comment by gran33** [comment]
Mmm, I'm pretty sure that the fix here isaacs/node-graceful-fs@a1587ea is what helped you. Your cache is fine this time :)

**Comment by alexsmartens** [comment]
I did not change package.json

**Comment by gran33** [comment]
Sure, it's one of react-native dependencies that just published a fix (this is the fix commit isaacs/node-graceful-fs@a1587ea).
